src/rgw/rgw_rados.cc

```diff
@@ -3764,6 +3764,22 @@ int RGWRados::init_rados()
     }
   }
 
+  {
```

Comment:
@yehudasa how can i tell if we are radosgw-admin and note that accordingly in the metadata?  

1. random cli commands shouldn't register
2. long-running radosgw-admin jobs maybe should, but they should appear as something different?